core/store/tx_manager.go

```diff
@@ -313,9 +327,10 @@ func (txm *EthTxManager) sendInitialTx(
 }
 
 var (
-	nonceTooLowRegex = regexp.MustCompile("(nonce .*too low|same hash was already imported)")
+	nonceTooLowRegex = regexp.MustCompile("(nonce .*too low|same hash was already imported|replacement transaction underpriced)")
 )
 
+// FIXME: There are probably other types of errors here that are symptomatic of a nonce that is too low
 func isNonceTooLowError(err error) bool {
 	return err != nil && nonceTooLowRegex.MatchString(err.Error())
 }
```

Comment:
Not-a-blocker:  We should check whether the package exports other ways to check errors besides string matching.  Personally I don't consider the error message a part of the API.  Atleast there should be integration tests to ensure the strings do not change no?